packages/zoe/src/issuerTable.js

```diff
@@ -39,19 +39,26 @@ const makeIssuerTable = () => {
       const brandP = E(issuerP).getBrand();
       const brandIssuerMatchP = E(brandP).isMyIssuer(issuerP);
       const amountMathKindP = E(issuerP).getAmountMathKind();
-      /** @type {[Issuer,Brand,boolean,AmountMathKind]} */
-      const [
-        issuer,
-        brand,
-        brandIssuerMatch,
-        amountMathKind,
-      ] = await Promise.all([
-        issuerP,
-        brandP,
-        brandIssuerMatchP,
-        amountMathKindP,
-      ]);
-      // AWAIT /////
+
+      let issuer;
+      let brand;
+      let brandIssuerMatch;
+      let amountMathKind;
+      try {
+        /** @type {[Issuer,Brand,boolean,AmountMathKind]} */
+        ([issuer, brand, brandIssuerMatch, amountMathKind] = await Promise.all([
```

Comment:
`await` inside a `try` violates our rules. It is silly in this case anyway. Please rephrase your catch clause as a rejection handler on the promise returned by `Promise.all`.